product_docs/docs/migration_portal/4.0/04_mp_migrating_database/03_mp_schema_migration.mdx

```diff
@@ -28,17 +28,25 @@ You can migrate schemas to an existing on-premises EDB Postgres Advanced Server
 1.  Select **Existing on-premises EDB Postgres Postgres Advanced Server**.
 
 
-2.  Select one or more schemas to migrate to EDB Postgres Advanced Server.
+1.  Select one or more schemas to migrate to EDB Postgres Advanced Server. You can also expand a schema and select a specific object types under each schema. 
+ 
+ ![Schema and objects are selected](../images/schema_object_selection.png)
 
     !!! Note
     If your schemas aren't 100% compatible, a banner appears. Complete the Contact Us form as needed.
 
 
-3.  To download the assessed schemas, select **Download SQL file**.
+1. To download the assessed schemas, select **Download SQL file**. 
 
+1. Extract the downloaded zip file. The zip package consists of SQL files with DDLs for your selection of schemas and objects. The SQL files group DDLs by project, schema, and object, and use the following naming convention. 
 
-4.  Select **Windows**.
+    - *<ProjectName>*_ALL_OBJECTS.sql: A single consolidated file consisting of all DDLs for your selected schemas and objects. 
```

Comment:
you can't use left angle brackets outside of code formatting unless you use the ascii code for them ( "&lt;" ). Looks like you have three instances that need to be changed. They are causing your deploys to fail.

Reply:
Thanks. i'll fix that.

Reply:
I edited my comment by adding spaces between the ascii code characters so they would show up.

Reply:
Fixed.